engines/kyra/detection_tables.h

```diff
@@ -316,23 +316,10 @@ const KYRAGameDescription adGameDescs[] = {
 		{
 			"kyra1",
 			"StuffIt multi-floppy",
-			AD_ENTRY1s("xn--Legend of Kyrandia Installer-o11r", "83d0f8e8c44a0aaa92b06081c40cd3c2", 69726),
+			AD_ENTRY1s("xn--Legend of Kyrandia Installer-o11r", "d:23ae9506979d844c29cccb40774e471e", 1363968),
```

Comment:
I thought that the md5 here is of the resource fork (with the ADGF_MACRESFORK flag). Was this tested?

Reply:
Yes. It had 2 entries: one to match proper dumps by resource fork and another one to match data fork if resource fork is missing. I switched to matching what we care about and what we use: data fork